pkg/genevalogging/genevalogging.go

```diff
@@ -322,12 +240,7 @@ func (g *genevaLogging) CreateOrUpdate(ctx context.Context) error {
 
 	g.log.Print("waiting for privileged security context constraint")
 	timeoutCtx, cancel := context.WithTimeout(ctx, 30*time.Minute)
-	defer cancel()
-	var scc *securityv1.SecurityContextConstraints
-	err = wait.PollImmediateUntil(10*time.Second, func() (bool, error) {
-		scc, err = g.seccli.SecurityV1().SecurityContextConstraints().Get("privileged", metav1.GetOptions{})
-		return err == nil, nil
-	}, timeoutCtx.Done())
+	scc, err := g.ensurer.Scc(timeoutCtx, cancel)
```

Comment:
not a good idea.  "ensurer" is basically a set of CreateOrUpdateFoo() functions made out of Create() and Update() functions (i.e. ensure Foo matches Bar).  This is adding a WaitForFooToExist() function (i.e. I don't care what Foo is but it must exist).  Let's not confuse the two things in the same package.

Reply:
Just so I'm sure I'm understanding this comment: generally, the ensurer is limited to create and update commands. In this case, it is also including a wait. We should refactor the ensurer to be limited to create and update commands, and handle the wait separately. Is that right?

Reply:
The wait can be handled with a `pkg/install/condition.go` function.  So instead of wrapping the ensurer comand with a wait, use the wait to either in the install/update functionality or outside of the ensurer.